src/apps/front-office/design-system/components/ui/select.tsx

```diff
@@ -0,0 +1,156 @@
+import {
+  CaretSortIcon,
+  CheckIcon,
+  ChevronDownIcon,
+  ChevronUpIcon,
+} from "@radix-ui/react-icons";
+import * as SelectPrimitive from "@radix-ui/react-select";
+import * as React from "react";
+
```

Comment:
Kindly remove any blank lines in the imports list unless there is a reason for it.

Reply:
i think this from shadcn ui but i will remove extra space now
and i need to mention is component is reason of "category arrows (category dropdown selection) are misplaced"